sfge/src/test/java/com/salesforce/rules/multiplemassschemalookup/MultipleMassSchemaLookupRuleTest.java

```diff
@@ -309,6 +308,31 @@ public void testLoopFromStaticBlock() {
         assertNoViolation(RULE, sourceCode);
     }
 
+    @Test
+    public void testLoopFromStaticBlock_nestedLoop() {
```

Comment:
Is this loop actually nested?

Reply:
Looks like a code merge issue. Good catch.